manifests/client.pp

```diff
@@ -1,7 +1,7 @@
 # Install client cli tool. See README.md for more details.
 class postgresql::client (
   Enum['file', 'absent'] $file_ensure        = 'file',
-  Stdlib::Absolutepath $validcon_script_path = $postgresql::params::validcon_script_path,
+  String[1] $validcon_script_path            = $postgresql::params::validcon_script_path,
```

Comment:
Note that this is a revert from upstream. The reason we revert this is including `Stdlib::Absolutepath` breaks compatibility with certain versions of puppet.

Reply:
which versions? Do you want to add a comment about the particular incompatibility?